_episodes/03-create.md

```diff
@@ -362,6 +368,12 @@ $ pwd
 ~~~
 $ mkdir thesis
 $ nano thesis/draft.txt
+~~~
```

Comment:
This is largely repetition of the above. I would think it best to avoid repeating in the lesson text even if you would repeat it verbally as an instructor?